packages/react-data-grid/src/Canvas.tsx

```diff
@@ -176,6 +166,13 @@ export default function Canvas<R, K extends keyof R>({
     }
   }, [rowRefs]);
 
+  useImperativeHandle(ref, () => ({
```

Comment:
That's going to update the ref on every render 🤔 
Hope no one uses a complicated ref function prop.